tomviz/python/AutoCrossCorrelationTiltImageAlignment.py

```diff
@@ -68,6 +68,10 @@ def transform_scalars(self, dataset):
 
         # Create a spreadsheet data set from table data
         column_names = ["X Offset", "Y Offset"]
+        offsets[np.where(offsets > tiltSeries.shape[0] / 2), :] = 
+            offsets[np.where(offsets > tiltSeries.shape[0] / 2), :] - tiltSeries.shape[0] / 2
+        offsets[:, np.where(offsets > tiltSeries.shape[1] / 2)] = 
+            offsets[:, np.where(offsets > tiltSeries.shape[1] / 2)] - tiltSeries.shape[1] / 2
```

Comment:
I think you want less than `-shape[0]/2` and greater than `shape[0]/2` (replace with shape[1] for Y) since you should also consider large negative shifts.  I'm also not sure what you are doing with the subtraction... wouldn't you want to set it to `0` or `shape[0]/2` rather than subtract an arbitrary amount?

Isn't the format of offsets [[x, y], [x, y], [x, y]], where the x and y are the offsets for that slice?  In that case you would want to fix the second coordinate to 0 or 1 based on which you want to set, and then always do your `np.where` in the first dimension.  As it is this will affect both the X and Y offsets if the X offset is bad and I think it will edit all offsets in the column (X or Y) if the first or second one is bad. Try this out with some simple numpy arrays on the python console to see if it does what you expect (or hardcode some bad values and see what the result is).